examples/default.rs

```diff
@@ -20,6 +20,9 @@ extern crate log;
 
 use env_logger::Env;
 
+#[cfg(all(target_arch = "wasm32", target_vendor = "unknown"))]
```

Comment:
Maybe we should add a dedicated example for wasm, even if it's almost exactly the same as this one, just to make it easier to see what's going on.